conda/recipes/libcudf_kafka/build.sh

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
+# This assumes the script is executed from the root of the repo directory
+
+# show environment
+printenv
+# Cleanup local git
+git clean -xdf
+# build libcudf with verbose output
+./build.sh -v libcudf_kafka --allgpuarch
```

Comment:
I don't think we can split this into a separate script like this. If we do so, it launches a separate conda build and the CMake isn't set up to detect an installed libcudf. This will end up building libcudf unnecessarily.

I think the best path forward would be to split the `cudf_kafka` library out into a separate CMake file which can then have a `find_library(cudf)` type call to find libcudf instead of depending on the cmake target.